### PR TITLE
v1.1.4 php 7 compatible

### PR DESCRIPTION
**Pull request by celtic34fr**
just some correction about plugins declaration, but the same as my last version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,7 @@
 E-mail module for Zend Framework 2
 ==================================
 
-[![Build Status](https://travis-ci.org/mtymek/MtMail.png?branch=master)](https://travis-ci.org/mtymek/MtMail)
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/mtymek/MtMail/badges/quality-score.png?s=f03d22161755c845d0ce06ab90a67cf4e3e340e0)](https://scrutinizer-ci.com/g/mtymek/MtMail/)
-[![Code Coverage](https://scrutinizer-ci.com/g/mtymek/MtMail/badges/coverage.png?s=d4b2ac636d43b3dd8042675dd32ac8fe2cf4e390)](https://scrutinizer-ci.com/g/mtymek/MtMail/)
-[![Latest Stable Version](https://poser.pugx.org/mtymek/mt-mail/v/stable.png)](https://packagist.org/packages/mtymek/mt-mail)
-[![Total Downloads](https://poser.pugx.org/mtymek/mt-mail/downloads.png)](https://packagist.org/packages/mtymek/mt-mail)
```
Comment (mtymek) [inline README.md]: Please revert this changes for the PR.

File: README.md [hunk 2]
```diff
@@ -44,7 +42,7 @@ Creating message from controller:
 $mailService = $this->getServiceLocator()->get('MtMail\Service\Mail');
 
 $headers = array(
-    'to' => 'johndoe@domain.com',
+    'to' => 'johndoe@domain.com, Jade DOE<jade.doe@domain.net>',
```
Comment (mtymek) [inline README.md]: Please revert to previous version.

File: composer.json [hunk 3]
```diff
@@ -1,7 +1,13 @@
 {
-    "name": "mtymek/mt-mail",
+    "name": "celtic34fr/mt-mail",
```
Comment (mtymek) [inline composer.json]: Please revert to previous version

File: src/ComposerPlugin/EmbeddingImages.php [hunk 4]
```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: gilbert
+ * Date: 21/10/16
+ * Time: 14:05
+ */
```
Comment (mtymek) [inline src/ComposerPlugin/EmbeddingImages.php]: Remove auto-generated doc-block - it does not bring any useful information.

File: README.md [hunk 5]
```diff
@@ -1,11 +1,7 @@
 E-mail module for Zend Framework 2
 ==================================
 
-[![Build Status](https://travis-ci.org/mtymek/MtMail.png?branch=master)](https://travis-ci.org/mtymek/MtMail)
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/mtymek/MtMail/badges/quality-score.png?s=f03d22161755c845d0ce06ab90a67cf4e3e340e0)](https://scrutinizer-ci.com/g/mtymek/MtMail/)
-[![Code Coverage](https://scrutinizer-ci.com/g/mtymek/MtMail/badges/coverage.png?s=d4b2ac636d43b3dd8042675dd32ac8fe2cf4e390)](https://scrutinizer-ci.com/g/mtymek/MtMail/)
-[![Latest Stable Version](https://poser.pugx.org/mtymek/mt-mail/v/stable.png)](https://packagist.org/packages/mtymek/mt-mail)
-[![Total Downloads](https://poser.pugx.org/mtymek/mt-mail/downloads.png)](https://packagist.org/packages/mtymek/mt-mail)
+It's a fork of mtymek/mt-mail package
```
Comment (ojhaujjwal) [inline README.md]: seriously?

**Review by mtymek — CHANGES_REQUESTED** [review]
I added a few comments here and there regarding some details. However, what I'm missing is:
- some explanation in the PR: what are the new features and why you introduced them
- documentation in `doc` directory
- unit tests for new features

File: src/ComposerPlugin/PlaintextMessage.php [hunk 6]
```diff
@@ -25,7 +25,22 @@ class PlaintextMessage extends AbstractListenerAggregate implements PluginInterf
      */
     private function br2nl($string)
     {
-        return preg_replace('/\<br(\s*)?\/?\>/i', "\n", $string);
+        /* suppression de la balise title */
```
Comment (ojhaujjwal) [inline src/ComposerPlugin/PlaintextMessage.php]: only english please

**Comment by ojhaujjwal** [comment]
Please add proper information in the PR description and the tests(mandatory). And some docs would be great as well.

**Comment by celtic34fr** [comment]
my latest changes : 
remove the french comments
add some information about new plugin EmbeddingImages and how to declare multiple recipients for an email

for UnitTest i don't see how to set it to be efficient. For PlaintextMessage plugin, my changes affects the message rendered before creating the mail in text format. For EmbeddingImages plugin, the UnitTest is difficult to set up as i know. Could help me in that case ?

**Comment by celtic34fr** [comment]
as I read, now MtMail is Php 7.2 compatible, this pulls request is full useless. I close it

**Pull request closed without merging**